src/main/java/com/example/auctionapp/controller/AddressController.java

```diff
@@ -0,0 +1,65 @@
+package com.example.auctionapp.controller;
+
+import com.example.auctionapp.dto.UserDtos.AddressDto;
+import com.example.auctionapp.service.AddressService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import javax.validation.Valid;
+import java.util.List;
+
+@RestController
+@RequestMapping("/address")
```

Comment:
Considering that these are tied to the user, i would suggest that the api endpoints be something like 'user/adress/', or 'user/id/adress/' etc.